lib/rdoc/generator/template/darkfish/css/rdoc.css

```diff
@@ -232,8 +226,7 @@ main {
 
 @media (min-width: 1024px) {
   main {
-    margin-left: auto;
-    margin-right: auto;
+    margin-left: 300px;
```

Comment:
Is 300px a specific value that must be used here and somewhere else? Is it the sidebar width, for example? It might be an opportunity to use a CSS (or SASS) variable instead.

Reply:
Yes it's `nav`'s width. I've added `--sidebar-width` variable for it.